lib/pure/parsecsv.nim

```diff
@@ -97,10 +96,10 @@ proc raiseEInvalidCsv(filename: string, line, col: int,
     e.msg = filename & "(" & $line & ", " & $col & ") Error: " & msg
   raise e
 
-proc error(my: CsvParser, pos: int, msg: string) =
-  raiseEInvalidCsv(my.filename, my.lineNumber, getColNumber(my, pos), msg)
+proc error(self: CsvParser, pos: int, msg: string) {.inline.} =
```

Comment:
You said that there was no logic change, yet you introduced `{.inline.}` here?

Reply:
Ye :shrug:

Reply:
Don't inline code that is off the critical path... Don't make error handling super fast, most of the time there is no error.